src/bandersnatch/verify.py

```diff
@@ -12,6 +12,7 @@
 from sys import stderr
 from typing import List, Optional, Set
 from urllib.parse import urlparse
+import aiohttp
```

Comment:
Pretty sure this just needs to move to make isort happy now.

Reply:
Done with "tox -e lint". Got there in the end. Been a busy few days.